nucypher/characters/chaotic.py

```diff
@@ -266,16 +266,21 @@ def home():
         @limiter.limit("5 per day")
         def register():
             """Handle new recipient registration via POST request."""
-            try:
-                new_address = request.form['address']
-            except KeyError:
-                return Response(status=400)  # TODO
+
+            # get address from form submission or json post
+            new_address = (
+                request.form.get('address') or
+                request.get_json().get('address')
+            )
+
+            if not new_address:
+                return Response(response="no address", status=400)  # TODO
```

Comment:
If you think that this is enough response data, Feel free to remove the `TODO`s.

Reply:
Yeah, I'm not sure.  I don't see why not other than maybe returning it in JSON format in the right situations.